changedetectionio/processors/text_json_diff.py

```diff
@@ -146,6 +146,7 @@ def run_changedetection(self, uuid, skip_when_checksum_same=True):
                 pass
 
         if has_filter_rule:
+            json_filter_prefixes = ['json:', 'jq:', 'jqraw:']
```

Comment:
https://github.com/dgtlmoon/changedetection.io/blob/d31fc860cc582b3a02569975777480f4b30baf47/changedetectionio/processors/text_json_diff.py#L21

you are re-defining this var, can you remove this line and just update that Line 21?

Reply:
done